src/notation/view/notationpaintview.cpp

```diff
@@ -567,7 +567,11 @@ void NotationPaintView::scale(qreal scaling, const QPoint& pos)
     int dx = pointAfterScaling.x() - pointBeforeScaling.x();
     int dy = pointAfterScaling.y() - pointBeforeScaling.y();
 
-    moveCanvas(dx, dy);
+    if (dx != 0 || dy != 0) {
```

Comment:
maybe we just have to put "update" at the beginning of the "moveCanvas", right before this condition?

```
if (dx == 0 && dy == 0) {
        return;
    }
```

 Otherwise, we can forget this logic the next time we need to use the moveCanvas method.

Reply:
I believe `moveCanvas` is correct if `dx` and `dy` are 0 then no need to move the canvas - no call update needed